### PR TITLE
Test if workflow change triggers test-suite

### PR DESCRIPTION
**Pull request by ifarzana**

### CODE REVIEW

**Comment by ifarzana** [comment]
TEST: all

**Comment by ifarzana** [comment]
TEST: all

**Comment by dasgarner** [comment]
This won't work until it gets into develop I think

**Comment by ifarzana** [comment]
Can you please approve and merge? Thanks

**Comment by dasgarner** [comment]
I think you still need to change the name of the status check that gets updated? Probably to "Build Containers and Run Tests" ?

File: .github/workflows/test-suite.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: Test Suite
+name: Build Containers and Run Tests
```
Comment (dasgarner) [inline .github/workflows/test-suite.yaml:1]: I think this should be the same and we should change the status update places.

Comment (ifarzana) [reply .github/workflows/test-suite.yaml:1]: `name: Build Containers and Run Tests`

Is this right?

Comment (ifarzana) [reply .github/workflows/test-suite.yaml]: Ah, I understand now. I don't have permission to edit the branch protection rule.

**Pull request merged**